plugin/src/plugin.ts

```diff
@@ -6,32 +6,51 @@ import type { ExplicitWorklet, ReanimatedPluginPass } from './types';
 import { processIfWorkletNode } from './processIfWorkletNode';
 import { processInlineStylesWarning } from './processInlineStylesWarning';
 import { processIfCallback } from './processIfCallback';
+import { isRelease } from './utils';
 
 module.exports = function (): PluginItem {
+  function callGuardDEV(fun: () => void) {
```

Comment:
I'm not fully convinced if we should use the name `callGuardDEV` here since it may be easily confused with `callGuardDEV` for running worklets.